pkg/crc/services/dns/dns_linux.go

```diff
@@ -12,7 +12,10 @@ func runPostStartForOS(serviceConfig services.ServicePostStartConfig, result *se
 	// We might need to set the firewall here to forward
 	// Update /etc/hosts file for host
 	if err := updateHostsConfFile(serviceConfig.IP, serviceConfig.BundleMetadata.GetAPIHostname(),
-		serviceConfig.BundleMetadata.GetAppHostname("oauth-openshift")); err != nil {
+		serviceConfig.BundleMetadata.GetAppHostname("oauth-openshift"),
+		serviceConfig.BundleMetadata.GetAppHostname("console-openshift-console"),
+		serviceConfig.BundleMetadata.GetAppHostname("default-route-openshift-image-registry"),
+		serviceConfig.BundleMetadata.GetAppHostname("foo")); err != nil {
```

Comment:
Why is this linux specific? We'll want exactly the same thing on all platforms.
I don't think we should add `foo` here, but rather we should use the test for `foo` to indicate whether dnsmasq-based DNS is working or not. This will be useful when debugging networking issues.